drivers/hud_pit/app/views/hud_pit/cells/show.xlsx.axlsx

```diff
@@ -2,12 +2,15 @@ wb = xlsx_package.workbook
 wb.add_worksheet(name: @name.slice(0, 30).gsub(':', ' ')) do |sheet|
   title = sheet.styles.add_style(sz: 12, b: true, alignment: { horizontal: :center })
   sheet.add_row(HudPit::Fy2022::PitClient.detail_headers_for_export.values, style: title)
+  current_user.policy_context.preload_project_dependencies(@clients.distinct.pluck(:project_id))
   @clients.preload(:source_client).each do |client|
+    pii_policy = GrdaWarehouse::AuthPolicies::LegacyPiiPolicy.instance if client.project_id.blank?
+    pii_policy ||= current_user.policy_for(client.project_id.to_i, policy_class: GrdaWarehouse::AuthPolicies::ProjectPiiPolicy)
```

Comment:
Let's leave this as is for now, but I wonder if it makes sense in some future version to create a method that returns either the legacy policy or the project policy.  Something like:
```
current_user.reporting_policy_for(project_id: client.project_id)
```
That could return `GrdaWarehouse::AuthPolicies::LegacyPiiPolicy.instance` if `project_id` is blank.  

It would DRY this up significantly and allow us to change the reporting policy behavior in one central location.